Dockerfile.model

```diff
@@ -1,24 +1,24 @@
 FROM php:{PHP_VERSION}-apache
+LABEL 'maintainer="Thomas Nabord <thomas.nabord@prestashop.com>"'
```

Comment:
The single quotes between `maintainer="Thomas Nabord <thomas.nabord@prestashop.com>"` can be removed.